rasa/cli/x.py

```diff
@@ -445,13 +456,19 @@ def run_locally(args: argparse.Namespace):
     process = start_rasa_for_local_rasa_x(args, rasa_x_token=rasa_x_token)
 
     config_path = _get_config_path(args)
+    domain_path = _get_domain_path(args)
 
     telemetry.track_rasa_x_local()
 
     # noinspection PyBroadException
     try:
         local.main(
-            args, project_path, args.data, token=rasa_x_token, config_path=config_path
+            args,
+            project_path,
+            args.data,
+            token=rasa_x_token,
+            config_path=config_path,
+            domain_path=domain_path,
```

Comment:
Unfortunately, `domain_path` argument is not yet supported by Rasa X.

Reply:
done in RasaHQ/rasa-x#3967